examples/all-clusters-app/Ameba/main/chipinterface.cpp

```diff
@@ -0,0 +1,69 @@
+#include <platform_stdlib.h>
+
+#include "CHIPDeviceManager.h"
+#include "DeviceCallbacks.h"
+#include "Server.h"
+
+#include <platform/CHIPDeviceLayer.h>
+#include <support/CHIPMem.h>
+
+extern "C"{ void * __dso_handle = 0 ;}
+
+using namespace ::chip;
+using namespace ::chip::DeviceManager;
+using namespace ::chip::DeviceLayer;
+
+namespace
+{
+    void SetupPretendDevices(void)
+    {
+        //TODO
+    }
+
+    class AppCallbacks : public AppDelegate
+    {
+    public:
+        void OnReceiveError() override { };
```

Comment:
nit: remove unnecessary semicolon

Reply:
Thanks for feedback. I removed null function which no support in the version.